Storage/src/Connection/Rest.php

```diff
@@ -750,9 +751,9 @@ private function updateRetryHeaders(
      *  header key.
      */
     private function updateHeader(
-        string $headerLine,
-        array &$arguments,
-        string $value,
+        $headerLine,
```

Comment:
Not part of this PR, but the docbloc here also could use the proper indentation.

Reply:
Updated in commit [ba03c51](https://github.com/googleapis/google-cloud-php/pull/5953/commits/ba03c51d90bb70cd400d755623229350d97ccd2d)